plugins/mongodb/mongodbembed.cpp

```diff
@@ -222,7 +222,7 @@ namespace mongodbembed
         }
         else
         {
-            failx("EJSON datatype error: %s is not supported in the current version.", key);
+            failx("EJSON datatype error: %s is not supported in the current version.", key.c_str());
```

Comment:
minor, but I find it helpful to encapsulate the %s placeholder in case key is empty or very long

Reply:
Do you mean something like this?
failx("EJSON datatype error: %s is not supported in the current version.", (!key.empty() || key.size() < 25) ? key.c_str() : "Unknown Datatype");

Reply:
That should be &&, but is that max length good? Should I throw another error all together?

Reply:
I just meant something like this:
failx("EJSON datatype error: '%s' is not supported in the current version.", key.c_str());

Reply:
Ok, much simpler.